File: android/filtering-content/GetItemsByItsLinkedItem.java
// DocSection: filtering_get_items_by_linked_item
// Gets items attributed to Jane. Matched items may also contain other authors.
List<ContentItem> items = deliveryService.<ContentItem>items()
    .containsFilter("elements.author", Arrays.asList("jane_doe"))
    .get()
    .getItems();

// Gets items attributed to at least Jane, John, or both. Matched items may also contain other authors.
List<ContentItem> items = deliveryService.<ContentItem>items()
    .anyFilter("elements.author", Arrays.asList("jane_doe", "john_wick"))
    .get()
    .getItems();
// EndDocSection

File: net/filter-content/GetItemsByItsLinkedItem.cs
// DocSection: filtering_get_items_by_linked_item
// Gets items attributed to Jane.
IDeliveryItemListingResponse<object> response = await deliveryClient.GetItemsAsync<object>(
    new ContainsFilter("elements.author", "jane_doe")
);

// Gets items attributed to at least Jane, John, or both.
IDeliveryItemListingResponse<object> response = await deliveryClient.GetItemsAsync<object>(
    new AnyFilter("elements.author", "jane_doe", "john_wick")
);
// EndDocSection